driver/main.cpp

```diff
@@ -11,13 +15,39 @@
 #include <nod/nod.hpp>
 
 static void printHelp() {
-  fmt::print(stderr, FMT_STRING(
+#if _WIN32
+  fmt::print(stderr, FMT_STRING(_SYS_STR(
           "Usage:\n"
           "  nodtool extract [-f] <image-in> [<dir-out>]\n"
           "  nodtool makegcn <fsroot-in> [<image-out>]\n"
           "  nodtool makewii <fsroot-in> [<image-out>]\n"
           "  nodtool mergegcn <fsroot-in> <image-in> [<image-out>]\n"
-          "  nodtool mergewii <fsroot-in> <image-in> [<image-out>]\n"));
+          "  nodtool mergewii <fsroot-in> <image-in> [<image-out>]\n"
+          "Options:\n"
+          "  -f         Force (extract only)\n"
+          "  -v         Verbose details (extract only).\n"
+          "  -c XXXXX   Set multi-byte character set of image(s).\n"
+          "Common Codepages for Windows:\n"
+          "  20127      7-bit ASCII (default)\n"
+          "  65001      UTF-8\n"
+          "  932        Shift-JIS\n")));
+#else
+  fmt::print(stderr, FMT_STRING(_SYS_STR(
+          "Usage:\n"
+          "  nodtool extract [-f] <image-in> [<dir-out>]\n"
+          "  nodtool makegcn <fsroot-in> [<image-out>]\n"
+          "  nodtool makewii <fsroot-in> [<image-out>]\n"
+          "  nodtool mergegcn <fsroot-in> <image-in> [<image-out>]\n"
+          "  nodtool mergewii <fsroot-in> <image-in> [<image-out>]\n"
+          "Options:\n"
+          "  -f         Force (extract only)\n"
+          "  -v         Verbose details (extract only).\n"
+          "  -c XXXXX   Set multi-byte character set of image(s).\n"
+          "Common Codepages for Linux:\n"
+          "  US-ASCII   7-bit ASCII (default)\n"
+          "  UTF-8      UTF-8\n"
+          "  SJIS       Shift-JIS\n")));
+#endif
```

Comment:
Any way we can unify this instead of having two different ways?

Reply:
http://site.icu-project.org/
Would it be worth using this library?  I think it can replace Windows stringapi and Linux's iconv.

Reply:
It uses a configure script rather than CMake, if you think you can get it cleanly integrated, I like the API better, and it's more sensibly organized.